### PR TITLE
prepating metrics

### PR DESCRIPTION
**Pull request by kiturutin**

### CODE REVIEW

File: lib/collectors/infrastructure_collector.rb [hunk 1]
```diff
@@ -1,21 +0,0 @@
-require 'aws_helper'
```
Comment (vkorobkov) [inline lib/collectors/infrastructure_collector.rb:1]: :+1:

File: lib/collectors/metric_collector.rb [hunk 2]
```diff
@@ -3,128 +3,149 @@
...
+    disks = samples[:disks]
+
+    cpu_usage = machine[:cpu_usage]
+    memory_bytes = machine[:memory_bytes]
+    network_in = nics[:network_in]
+    network_out = nics[:network_out]
+
+    @timestamps.each do |time|
+      machine_sample =
+          MachineSample.new(cpu_usage_percent: cpu_usage[time],
```
Comment (vkorobkov) [inline lib/collectors/metric_collector.rb:92]: Where is instance storages consumption?

File: lib/models/inventory/host.rb [hunk 3]
```diff
@@ -11,9 +11,10 @@ class Host
   field :launch_time, type: DateTime
   field :monitoring, type: String
 
-  field :memory_gb,  type: Float
+  field :memory_gb, type: Float
   field :network, type: String
   field :platform, type: String
+  field :root_device_type, type: String, default: 'instance-store'
```
Comment (vkorobkov) [inline lib/models/inventory/host.rb]: Where inventory populates this field? Why we need a default value?

File: lib/collectors/metric_collector.rb [hunk 4]
```diff
@@ -3,128 +3,149 @@
...
-  ]
-
-  def process(options)
-    initialize_metric_collection(options[:start_time])
-
-    common_options = merge_common_options(options)
-    METRICS.each do |metric|
-      process_metric(metric, common_options)
-    end
+  MEGABYTES_TO_BYTES = ->(value) { value.megabytes }
```
Comment (vkorobkov6) [inline lib/collectors/metric_collector.rb]: Who needs it?

File: lib/models/inventory/inventory.rb [hunk 5]
```diff
@@ -1,10 +1,10 @@
 class Inventory
   include Mongoid::Document
+  include Mongoid::Timestamps::Created
```
Comment (vkorobkov6) [inline lib/models/inventory/inventory.rb:3]: :+1:

**Pull request merged**